lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -229,8 +237,8 @@ def unsafe_remove_constraint(table, name:)
   def safe_create_partitioned_table(table, key:, type:, infer_primary_key: nil, **options, &block)
     raise ArgumentError, "Expected <key> to be present" unless key.present?
 
-    unless VALID_PARTITION_TYPES.include?(type)
-      raise ArgumentError, "Expected <type> to be symbol in #{VALID_PARTITION_TYPES}"
+    unless PARTITION_TYPES.include?(type)
+      raise ArgumentError, "Expected <type> to be symbol in #{PARTITION_TYPES}"
```

Comment:
This code isn't actually relevant to this PR, but seems like a reasonable suggestion. There's a diff here because I renamed `VALID_PARTITION_TYPES` to `PARTITION_TYPES` to be more consistent with the other constant that was added in this PR.